lib/Makefile

```diff
@@ -74,7 +74,7 @@ endif
 	$(INSTALL) ../include/mosquitto.h "${DESTDIR}${incdir}/mosquitto.h"
 	$(INSTALL) ../include/mqtt_protocol.h "${DESTDIR}${incdir}/mqtt_protocol.h"
 	$(INSTALL) -d "${DESTDIR}${libdir}/pkgconfig"
-	$(INSTALL) -m644 ../libmosquitto.pc.in "${DESTDIR}${libdir}/pkgconfig/libmosquitto.pc"
+	$(INSTALL) -m64 ../libmosquitto.pc.in "${DESTDIR}${libdir}/pkgconfig/libmosquitto.pc"
```

Comment:
Is this a mistake? On Linux that is equivalent to installing a `-m064` which is a very unhelpful set of permissions.